signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java

```diff
@@ -199,6 +203,12 @@ public boolean isEmpty() {
         return mIsEmpty;
     }
 
+    public String getSignatureSvg() {
+        int width = getTransparentSignatureBitmap().getWidth();
+        int height = getTransparentSignatureBitmap().getHeight();
+        return Svg.header(width, height) + mSvgBuilder.toString() + Svg.footer();
```

Comment:
In that case, as per my previous comments, here you should call `svgCurve.build()` to get the SVG XML string.